$MirOS: ports/graphics/libgphoto2/patches/patch-libgphoto2_libgphoto2_pc_in,v 1.1.7.1 2005/03/18 15:46:59 tg Exp $
$OpenBSD: patch-libgphoto2_libgphoto2_pc_in,v 1.1.1.1 2003/04/09 23:55:01 espie Exp $
--- libgphoto2/libgphoto2.pc.in.orig	Fri Jun 11 15:03:57 2004
+++ libgphoto2/libgphoto2.pc.in	Thu Dec 29 21:26:53 2005
@@ -9,5 +9,5 @@ Name: libgphoto2
 Description: Library for easy access to digital cameras
 Requires:
 Version: @VERSION@
-Libs: -L${libdir} -lgphoto2 -lgphoto2_port -lm
+Libs: -L${libdir} -lgphoto2 -lm -lgphoto2_port -lexif
 Cflags: -I${includedir}/gphoto2 -I${includedir}
